proto.go

```diff
@@ -13,10 +13,10 @@ const (
 	// See: https://developers.google.com/protocol-buffers/docs/proto
 	Proto2 Syntax = ""
 
-	// Proto3 syntax only allows for optional fields, but defaults to the zero
-	// value of that particular type. Most of the field types in the generated go
-	// structs are value types.
-	// See: https://developers.google.com/protocol-buffers/docs/proto3
+	// Proto3 syntax permits the use of "optional" field presence. Non optional fields default to the zero
```

Comment:
I think this explanation slightly confuses optionality as the opposite of requiredness versus optionality as having explicit presence. Even with explicit presence it is true that `Proto3 syntax only allows for optional fields`.

I would recommend rewording to something like 
`Proto3 syntax only allows for optional fields, and it defaults to the zero value of that particular type (unless experimental support for explicit presence tracking for singular proto3 fields is used, see <link>). `

Reply:
`(unless experimental support for explicit presence..`

According to [protobuf v3.15.0](https://github.com/protocolbuffers/protobuf/releases/tag/v3.15.0) field presence support in proto3 is no longer experimental.